docs_zh-CN/dataset_prepare.md

```diff
@@ -195,37 +195,3 @@ python tools/convert_datasets/stare.py /path/to/stare-images.tar /path/to/labels
 ### Nighttime Driving
 
 因为我们只支持在此数据集上测试模型，所以您只需下载[测试集](http://data.vision.ee.ethz.ch/daid/NighttimeDriving/NighttimeDrivingTest.zip)。
-
-### LoveDA
-
-下载[LoveDA数据集](https://drive.google.com/drive/folders/1ibYV0qwn4yuuh068Rnc-w4tPi0U0c-ti?usp=sharing)。
-
-对于 LoveDA 数据集，请运行以下命令下载并组织数据集
-
-```shell
-
-# download
-
-mkdir loveda && cd loveda
-wget https://drive.google.com/drive/folders/1ibYV0qwn4yuuh068Rnc-w4tPi0U0c-ti?usp=sharing
-
-# unzip
-
-unzip '*.zip'
-
-# Convert into segmentation splits
-
-mkdir -p img_dir/train img_dir/val img_dir/test ann_dir/train ann_dir/val
-mv Train/Rural/images_png/* img_dir/train
-mv Train/Urban/images_png/* img_dir/train
-mv Val/Rural/images_png/* img_dir/val
-mv Val/Urban/images_png/* img_dir/val
-mv Test/Rural/images_png/* img_dir/val
-mv Test/Urban/images_png/* img_dir/val
-mv Train/Rural/masks_png/* ann_dir/train
-mv Train/Urban/masks_png/* ann_dir/train
-mv Val/Rural/masks_png/* ann_dir/val
-mv Val/Urban/masks_png/* ann_dir/val
-```
-
-关于 LoveDA 的更多细节可以在[这里](https://github.com/Junjue-Wang/LoveDA)找到。
```

Comment:
Why remove this description?

Reply:
Sorry for this confusion. 
Because it is really difficult for me to install the pre-commit environment (Incompatible system, old version). So I manually revised the entire catalog with the mentioned tools in https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md.
Please help me check the lint errors in the last version. Thank you very much!

> @Junjue-Wang It seems that your pre-commit behaviors are not consistent with ours which leads to too many files being changed. @MengzhangLI Could you please fix it?

Reply:
> Why remove this description?

This is my mistake, please reject this push.

Reply:
You could push it again.

Reply:
> You could push it again.

I have pushed the previous version again. Please check.

Reply:
Great.